### PR TITLE
cleanup: standardize on oci_string and helm_release values

### PR DESCRIPTION
**Pull request by imjasonh**

### CODE REVIEW

File: images/cert-manager/main.tf [hunk 1]
```diff
@@ -50,7 +50,7 @@ module "test-latest" {
 module "test-latest-dev" {
   source = "./tests"
 
-  digests   = { for k, v in module.latest : k => v.image_ref }
```
Comment (imjasonh) [inline images/cert-manager/main.tf:53]: 👀

**Review by mattmoor — APPROVED** [review]

File: images/configmap-reload/tests/main.tf [hunk 2]
```diff
@@ -23,20 +25,13 @@ resource "helm_release" "dex" {
...
-    name  = "configmapReload.enabled"
-    value = "true"
-  }
+  values = [jsonencode({
+    configmapReload = {
+      image = {
+        tag        = data.oci_string.ref.pseudo_tag
+        repository = data.oci_string.ref.registry_repo
+      }
+      enabled = "true"
```
Comment (mattmoor) [inline images/configmap-reload/tests/main.tf:34]:
```suggestion
      enabled = true
```

**Review by mattmoor — DISMISSED** [review]

**Review by mattmoor — DISMISSED** [review]

**Review by mattmoor — DISMISSED** [review]

File: images/kyverno/tests/main.tf [hunk 3]
```diff
@@ -30,73 +35,49 @@ resource "helm_release" "kyverno" {
...
-  }
+  values = [jsonencode({
+    admissionController = {
+      container = {
+        image = {
+          registry = data.oci_string.ref["admission"].registry
+          registry = data.oci_string.ref["admission"].repo
+          tag      = data.oci_string.ref["admission"].pseudo_tag
+        }
+      }
```
Comment (mattmoor) [inline images/kyverno/tests/main.tf:46]: weird:
```
Failed to pull image "testing/kyvernopre/kyverno/kyvernopre:unused@sha256:1890a93052701b99b23cf4f015ab028f42b8d5ab280649fa116e125623913780"
```

Comment (mattmoor) [reply images/kyverno/tests/main.tf:46]: oh this is `initContainer` so below

**Review by mattmoor — APPROVED** [review]

**Pull request merged**